ci-operator/config/openshift/ci-tools/openshift-ci-tools-master.yaml

```diff
@@ -83,6 +88,14 @@ images:
       - destination_dir: .
         source_path: /go/bin/determinize-ci-operator
   to: determinize-ci-operator
+- context_dir: images/ipi-deprovision/
```

Comment:
I think this stanza needs to be simply:

```
- dockerfile_path: images/ipi-deprovision/Dockerfile
  from: hive
  to: ipi-provisioner
```

(provided https://github.com/openshift/ci-tools/pull/26#discussion_r305897546 is addressed)

Reply:
done.